hw-kafka-client.cabal

```diff
@@ -1,7 +1,7 @@
 cabal-version: 2.2
 
 name:                   hw-kafka-client
-version:                5.0.0
+version:                5.1.0
```

Comment:
Can you please rollback this? 
Otherwise it will mess with the release workflow

Reply:
Done.